Source/WebCore/PAL/pal/PALSwift.h

```diff
@@ -28,8 +28,39 @@
 #include <cstdint>
 #include <wtf/Vector.h>
 
+namespace Cpp {
```

Comment:
Are we trying to resolve a namespace conflict, or something else?

Reply:
Trying to do two things here:
1. Keep all the wrapper code inside a namespace and not pollute the Global namespace.
2. Use a nested namespace `Unsafe` to keep the bits we would like to get rid of when possible.